nmos-testing-facade.py

```diff
@@ -12,21 +12,23 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import random
 import requests
 import json
 import time
 from flask import Flask, render_template, make_response, request
 from flask_socketio import SocketIO
-from DataStore import data
+from testingfacade.DataStore import data
+from nmostesting import Config as CONFIG
 
-app = Flask(__name__)
+base_dir = os.path.abspath('testingfacade')
+app = Flask(__name__, static_folder=base_dir+"/static", template_folder=base_dir+"/templates")
```

Comment:
Did you try it like this?

```suggestion
app = Flask(__name__, root_path='testingfacade')
```

Reply:
Or if that didn't work, with relative paths like this...

```suggestion
app = Flask(__name__, static_folder='testingfacade/static', template_folder='testingfacade/templates')
```

Reply:
`root_path` didn't work, it couldn't find the static files but the relative paths worked. I tried those initially and it wasn't working, wasn't till after I changed to using `base_dir` that I realised for some reason there were 2 `app = Flask(__name__)` lines and the second was overriding the values I set in the first. Not sure when/how that got repeated